src/test/java/guitests/PersonEditGuiTest.java

```diff
@@ -52,21 +51,20 @@ public void editPerson_usingAccelerator() {
         assertEquals(alicePersonCard, newAlice);
         assertTrue(personListPanel.isSelected(newAlice));
 
-
         //Confirm right values are displayed after grace period is over
         sleepForGracePeriod();
         assertEquals(alicePersonCard, newAlice);
 
+        //Confirm cancel operation does not cancel the edit after the grace period.
+        personListPanel.use_PERSON_CHANGE_CANCEL_ACCELERATOR();
```

Comment:
Right, personListPanel.getSelectedPerson().toString() gives the backing person details. I think it is not appropriate to test it in guitest. Checking if the GUI details match is the one we should test in GUITest.